packages/create-cloudflare/src/helpers/command.ts

```diff
@@ -26,7 +26,7 @@ export const runCommand = async (
 		s.start(opts?.startText || command);
 	}
 
-	const [executable, ...args] = command.split(" ");
+	const [executable, ...args] = command.trim().split(" ");
```

Comment:
What happens if someone leaves multiple spaces between args? Will that break anything too?

Reply:
We really need to start writing tests to stop regressions like this.

Reply:
Agreed. I've added a unit test to cover this particular corner case, but the real safety net we needed here was an e2e test that exercises the deployment logic, which I've been putting off. I'll make an issue to track it.